xml_inscriptions/transcribed/PA.Phil.UP.UM.L.L-1033-51.xml

```diff
@@ -152,14 +155,17 @@
+                <lb n="1"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erpernae</name></persName> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>iberti</ex></expan> <unclear reason="lost">N(?)</unclear> <gap reason="lost" unit="character" extent="unknown"/>
                 <lb n="2"/> ossa hic sita su<supplied reason="lost">nt</supplied>.
-                <lb n="3"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arcus</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erperna</name> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>iberti</ex></expan> <name type="cognomen">Phila<gap reason="lost" unit="character" extent="unknown"/></name></persName>
+                <lb n="3"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arcus</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erperna</name> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>ibertus</ex></expan> <name type="cognomen">Phila<unclear reason="lost">d</unclear><gap reason="lost" unit="character" extent="unknown"/></name></persName>
                 <lb n="4"/> politor eborariu<supplied reason="lost">s</supplied>.
             </ab></div>
+            
+            <div type="translation">
+                <p>The bones of Marcus Perperna, freedman of Marcus, are situated here. Marcus Perperna Phila[   ], polisher of and worker in ivory, (dedicated this).</p></div>
 
             <div type="commentary">
-            <p>Notes here, </p></div>
+            <p>Line 1 breaks off with an unclear letter, with a vertical stroke on the left being all that is preserved; an "N" is supposed here which presumably began the individual's cognomen. While the Penn Museum treats "politor eborariu[s]" as a single vocational epithet, namely "ivory polisher," I chose instead to represent it as two independent epithets given the attestation of "eborarius" being used independently (in Justinian's codex, for instance) to refer to a class of ivory workers.</p></div>
```

Comment:
Remove some of the personal narrative here as to why you made the choice you did; just say that they do it one way, however eborarius is attested elsewhere etc.